common/src/main/java/org/valkyrienskies/mod/mixin/client/world/MixinClientChunkCache.java

```diff
@@ -79,8 +85,10 @@ private void preLoadChunkFromPacket(final int x, final int z, final ChunkBiomeCo
     @Inject(method = "drop", at = @At("HEAD"), cancellable = true)
     public void preUnload(final int chunkX, final int chunkZ, final CallbackInfo ci) {
         shipChunks.remove(ChunkPos.asLong(chunkX, chunkZ));
-        ((IVSViewAreaMethods) ((LevelRendererAccessor) ((ClientLevelAccessor) level).getLevelRenderer()).getViewArea())
-            .unloadChunk(chunkX, chunkZ);
+        if (ValkyrienCommonMixinConfigPlugin.getVSRenderer() != VSRenderer.SODIUM) {
```

Comment:
Do we need to call something similar for Sodium as well?

Reply:
fixed